test/Microsoft.AspNet.Mvc.Formatters.Json.Test/JsonInputFormatterTest.cs

```diff
@@ -187,6 +195,13 @@ public static IEnumerable<object[]> JsonFormatterReadSimpleTypesData
             Assert.True(result.HasError);
             Assert.Equal("The supplied value is invalid for Byte.", modelState["[2]"].Errors[0].ErrorMessage);
             Assert.Null(modelState["[2]"].Errors[0].Exception);
+            Mock.Get<ILogger>(logger).Verify(l => l.Log(
+                LogLevel.Verbose,
+                It.IsAny<int>(),
+                It.IsAny<object>(),
+                It.IsAny<Exception>(),
+                It.IsAny<Func<object, Exception, string>>()),
+            Times.Once);
```

Comment:
Maybe just fix the indentation then? The `Times.Once` can't be on the same line as `Mock.Get`.

Try something like:

```c#
Mock
    .Get<ILogger>(logger)
    .Verify(l => l.Log(  
        LogLevel.Verbose,  
        It.IsAny<int>(),  
        It.IsAny<object>(),  
        It.IsAny<Exception>(),  
        It.IsAny<Func<object, Exception, string>>()),  
    Times.Once);  
```